src/frontends/pytorch/src/op/convnd.cpp

```diff
@@ -50,7 +50,7 @@ OutputVector translate_convnd(const NodeContext& context) {
                                                       pad_type);
     }
     if (!context.input_is_none(2)) {
-        auto bias = context.get_input(2);
+        auto bias = context.get_input_from_visible_context(2);
```

Comment:
We can do this only when bias is constant. Please add such verification.

Reply:
Added cast to Constant as condition to check whether bias can be acquired from visible context